project_task_analytic_tag/README.rst

```diff
@@ -0,0 +1,85 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+Project Task Analytic Tag
+=========================
+
+This module extends the functionality of project to support to
+add Tags in the timesheet and allow you to use the tags of the projects 
+in your timesheet
+
+Installation
+============
+
+To install this module, you need to:
+
+- First Add in the addons path the module and install in your odoo.
+  And install the dependencies of the module
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+- You only need to have all the dependencies installed.
+
+Usage
+=====
+
+To use this module, you need to:
+
+- Go to project and enter in a task section, here you only need to go to the
```

Comment:
Really no more code is needed, now on tasks, timesheets page are analytic entries (previously are timesheet_sheet_lines that are related).
Then this module just show tag_ids from analytic_account_line on project_task. And set analytic_tag_ids on project task to define default analytic_tag_ids for all timesheets (account_analytic_line) for this task.